slack/src/code.gs

```diff
@@ -115,13 +149,18 @@ function getSchema(request) {
  */
 
 function getData(request) {
+  var requestedFieldIds = request.fields.map(function(field) {
+    return field.name;
+  });
+  var requestedFields = getFields().forIds(requestedFieldIds);
+  
   var url = [
     'https://slack.com/api/channels.history?token=',
     request.configParams.token,
     '&channel=',
     request.configParams.channel,
     '&count=',
-    request.configParams.count,
```

Comment:
Shouldn't this still use the configParam?

Reply:
That's done